tabcmd/commands/datasources_and_workbooks/publish_command.py

```diff
@@ -74,8 +74,8 @@ def run_command(args):
 
             new_workbook = TSC.WorkbookItem(project_id, name=args.name, show_tabs=args.tabbed)
             try:
-                if creds:
-                    logger.debug("Workbook credentials object: " + creds)
+                # if creds:
+                #     logger.debug("Workbook credentials object: " + creds)
```

Comment:
@jacalata I commented this out because it would throw an error saying creds is not a string type so it cannot append to string. This only errored when trying to publish the workbook with an embedded connection. How should I try to fix this?

Reply:
Probably delete it, we don't want to log credentials by default.